src/inputs/ObjectSearchCreate.tsx

```diff
@@ -96,23 +96,25 @@ class ObjectSearchCreate extends Component<IObjectSearchCreateProps> {
     const { fieldConfig, formManager } = this.injected;
 
     return (
-      <>
-        <NestedFieldSet
-          fieldSet={this.fieldConfig.createFields}
-          formManager={formManager}
-          formModel={formManager.formModel}
-          id={fieldConfig.field}
-          label={renderLabel(this.fieldConfig)}
-          search={this.search}
-        />
-        <Antd.Button
-          className={`${CX_PREFIX_SEARCH_CREATE}-btn-back`}
-          onClick={this.onSearch}
-          size='small'
-        >
-          <Antd.Icon type='left' /> Back to search
-        </Antd.Button>
-      </>
+      <Antd.Col>
```

Comment:
what is the antd.col and why is it wrapped in it?

Reply:
I was mostly copying the wrapping provided by `FormItem.tsx` which wraps all our inputs:
![image](https://user-images.githubusercontent.com/796717/58585698-65841080-8227-11e9-89bf-58e471c7e504.png)

`Col` is a Column layout component in Ant Design. I don't think it matters hugely here but I like the consistency with `FormItem`.

Reply:
Did you add it just for consistency or also because it fixed the spacing issue?

Reply:
The spacing issue was fixed by `Form.Item`. The Col was added because it matched `FormItem.tsx` and I think having one be `Col > Form.Item` and the other being just `Form.Item` introduces an inconsistency I don't like.

Reply:
Got it!